### PR TITLE
ONXP-1440 - [Gideon][BlockChain] Agent should pay 800$ instead of 500$ and get 500 OnyxCash

### PR DESCRIPTION
**Pull request by OnyxPayDev**
Changed text and fee for upgrading to Agent.

image

### CODE REVIEW

File: src/pages/upgrade-user/index.js [hunk 1]
```diff
@@ -175,7 +175,7 @@ class UpgradeUser extends Component {
 	};
 
 	getStepComponent(role) {
-		const paymentAmount = role === "agent" ? 500 : 100000;
+		const paymentAmount = role === "agent" ? 800 : 100000;
```
Comment (arsonaut) [inline src/pages/upgrade-user/index.js]: Magic constant literals. Does project has better place to declare and name them?

Comment (akorunska) [reply src/pages/upgrade-user/index.js]: If we decide to do so, it's nice to move those values to api/contants.js

Comment (OnyxPayDev) [reply src/pages/upgrade-user/index.js]: Changed to constant declarations from api/constants
Added declaration for payment amount

File: src/pages/upgrade-user/index.js [hunk 2]
```diff
@@ -175,7 +175,7 @@ class UpgradeUser extends Component {
 	};
 
 	getStepComponent(role) {
-		const paymentAmount = role === roles.a ? 800 : 100000;
+		const paymentAmount = paymentAmountByRole[role];
```
Comment (akorunska) [inline src/pages/upgrade-user/index.js]: I can't see the appropriate update in contants.js file. Bug or feature?)

File: src/pages/upgrade-user/index.js [hunk 3]
```diff
@@ -33,12 +33,26 @@ function getStepTitle(item, step) {
 }
 
 function getTitleRoleByRole(role) {
```
Comment (akorunska) [inline src/pages/upgrade-user/index.js]: Please, use verbose user role functionality from the utils instead of declaring it directly in the component.

**Pull request merged**